COVIDWatch iOS/ViewControllers/Home.swift

```diff
@@ -17,12 +17,28 @@ class Home: BaseViewController {
     var testedButton = Button(text: "Tested for COVID-19?",
                               subtext: "Share your result anonymously to help keep your community stay safe.")
     var infoBanner = InfoBanner(text: "You may have been in contact with COVID-19")
+    var lastTestedDateObserver: NSKeyValueObservation?
+    var didUserMakeContactWithSickUserObserver: NSKeyValueObservation?
+
     override func viewDidLayoutSubviews() {
         super.viewDidLayoutSubviews()
+        lastTestedDateObserver = UserDefaults.shared.observe(
+            \.lastTestedDate,
+            options: [.initial, .new],
+            changeHandler: { (_, _) in
+            self.drawScreen()
+        })
+
+        didUserMakeContactWithSickUserObserver = UserDefaults.shared.observe(
+            \.didUserMakeContactWithSickUser,
+            options: [],
+            changeHandler: { (_, _) in
+            self.drawScreen()
+        })
```

Comment:
Does observation registration need to happen when the views are laid out? Or can this be moved to viewDidLoad?

Reply:
Good point, it only needs to happen once.